deploy/gcp/values.yaml

```diff
@@ -27,7 +27,7 @@
 replicaCount: 1
 
 image:
-  imageName: nvcr.io/nvidia/tritonserver:22.02-py3
+  imageName: nvcr.io/nvidia/tritonserver:22.04-py3
```

Comment:
Do we always advance this @dzier ?

Was this not advanced for 22.03 intentionally?

Reply:
The `values.yaml` is used by HELM chart and designed to pull proper image from repository during the object deployment in Kubernetes. I assume it must be up to date with our development version.

Reply:
Update the copyright for this file